src/main/scala/advancepassengerinfo/health/ProcessState.scala

```diff
@@ -0,0 +1,16 @@
+package advancepassengerinfo.health
+
+import java.time.{Duration, Instant}
+
+case class ProcessState() {
+   var healthState: Instant = Instant.now()
```

Comment:
Rather than create another place where we have the time the last file was processed, we can look up the latest `processed_at` from the `processed_zip` table